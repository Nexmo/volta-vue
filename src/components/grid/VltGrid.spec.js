import { h } from 'vue';
import { mount } from '@vue/test-utils';
import { VltCol, VltGrid } from '..';

describe('vlt-grid', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount({
      render() {
        return h(VltGrid, {}, [h(VltCol), h(VltCol), h(VltCol)]);
      },
    });
  });

  test('renders correctly with three columns', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
