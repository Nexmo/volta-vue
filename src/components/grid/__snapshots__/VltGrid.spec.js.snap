// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`vlt-grid renders correctly with three columns 1`] = `
<div
  class="Vlt-grid"
>
  
  <div
    class="Vlt-col"
  >
    
    
  </div>
  <div
    class="Vlt-col"
  >
    
    
  </div>
  <div
    class="Vlt-col"
  >
    
    
  </div>
  
</div>
`;
